cmd/examples/example1.go

```diff
@@ -46,12 +46,24 @@ func main() {
 		CPUArchitecture: &cpuArch,
 	}
 
-	// Pass the Filter struct to the Filter function of your selector instance
-	instanceTypesSlice, err := instanceSelector.Filter(filters)
+	// Pass the Filter struct to the GetFilteredInstanceTypes function of your
+	// selector instance to get a list of filtered instance types and their details
+	instanceTypesSlice, err := instanceSelector.GetFilteredInstanceTypes(filters)
```

Comment:
`FilterInstanceTypes` was the name that I had originally used, but I felt that it might be slightly confusing since the function `GetFilteredInstanceTypes` both fetches and then filters instance types instead of just filtering instance types. With this context in mind, I feel as though `Get` adds the proper context.

Reply:
In the context of the instance-selector lib, the `Get` part is irrelevant since the lib is abstracting how that is accomplished. From a user's perspective, the Filtering process isn't a two-step process, it's simply "give me a filtered list of instance types" (and even in the actual implementation it's more like  3 or 4 step process). In general, `Get` is a filler word that doesn't add much extra information and isn't very common in go.